src/SketchField.jsx

```diff
@@ -446,7 +459,10 @@ class SketchField extends PureComponent {
    *
    * @returns {String} URL containing a representation of the object in the format specified by options.format
    */
-  toDataURL = (options) => this._fc.toDataURL(options);
+  toDataURL = (options, targetWatershed = false) => {
+    const targetCanvas = targetWatershed ? this._watershedCanvas : this._fc;
```

Comment:
Shouldn't this be:
```suggestion
    const targetCanvas = targetWatershed ? this._fwc : this._fc;
```
Not sure though, but since the other option of the ternary expression is `this._fc`, which is a fabric canvas, I got confused (I'm aware that HTML Canvas has the `toDataURL` method as well).

Reply:
Yes, but we are adding the contents to the canvas via opencv, directly